datumaro/plugins/vgg_face2_format.py

```diff
@@ -130,15 +141,21 @@ def _split_item_path(path):
 class VggFace2Importer(Importer):
     @classmethod
     def find_sources(cls, path):
-        return cls._find_sources_recursive(path, '.csv', 'vgg_face2',
-            dirname=VggFace2Path.ANNOTATION_DIR,
-            file_filter=lambda p: \
-                not osp.basename(p).startswith(VggFace2Path.BBOXES_FILE))
+        annotation_dir = osp.join(path, VggFace2Path.ANNOTATION_DIR)
+        if osp.isdir(annotation_dir):
+            return [{'url': annotation_dir, 'format': 'vgg_face2'}]
+        return []
```

Comment:
Now it is impossible to load a specific subset.

Reply:
I pushed new commits, now user can load the vgg_face2 dataset in two ways: 1) from directory with dataset (all subsets will be load) 2) from specific annotation file (only specified subset will be load)